invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/invitations/InvitationActionButtons.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Button } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_communities/i18next";
+
+export const ViewButton = ({ onClick }) => (
+  <Button size="tiny" content={i18next.t("View")} onClick={onClick} />
+);
+
+export const CancelButton = ({ onClick }) => (
+  <Button size="tiny" content={i18next.t("Cancel")} onClick={onClick} />
+);
+
+export const ReInviteButton = ({ onClick }) => (
+  <Button content={i18next.t("Re invite")} onClick={onClick} />
+);
+
+export const ActionButtons = ({ status, onReInvite, onView, onCancel }) => {
+  const condition = false;
```

Comment:
If condition is a parameter it could be more reusable.

Reply:
todo